web/templates/popovers/user_group_info_popover.hbs

```diff
@@ -26,23 +26,67 @@
         <li role="separator" class="popover-menu-separator"></li>
         <li role="none" class="popover-menu-list-item">
             {{#if (or members.length subgroups.length)}}
-                <ul class="popover-menu-list popover-group-menu-member-list" data-simplebar data-simplebar-tab-index="-1" data-simplebar-auto-hide="false">
-                    {{#each subgroups}}
-                        <li class="popover-group-menu-member">
-                            <i class="popover-group-member-icon popover-menu-icon zulip-icon zulip-icon-triple-users" aria-hidden="true"></i>
-                            <span class="popover-group-menu-member-name">{{name}}</span>
-                        </li>
+                <ul class="popover-menu-list popover-group-menu-member-list" data-simplebar
+                  data-simplebar-tab-index="-1" data-simplebar-auto-hide="false">
+                    {{#each displayed_members_and_subgroups}}
+                        {{#if user_id}}
```

Comment:
I think would be good to have different `displayed_members` and `displayed_subgroups` list so that we can avoid having this `#if user_id` check.